### PR TITLE
Update images and doc user manual

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for issue #1681 

@LunaAsefaw task one's image is already updated.

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 37 sec

File: docs/src/user/manual/explore/filters.md [hunk 1]
```diff
@@ -304,20 +329,32 @@ The filter panel empowers users to customise their data views efficiently, allow
...
+#### 3. Province Filter
+
+* **Filter Description:** The province filter helps users filter data based on provinces. This filter allows users to make multiple selections for a more customised view.
+* **Usage:** Select a province from the available options to view data related to that province.
+
+    ![Province filter](./img/filters-20.png)
 
-* **Filter Description:** The property filter helps users filter data based on properties. This filter is dependent on the organisation filter.
-It means the options will be based on the selected organisations.
+1. **Province Filter:** In the province filter, all provinces are initially selected as the default setting. Users have the freedom to hand-pick a provice of their choice.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/filters.md:347]:
```suggestion
1. **Province Filter:** In the province filter, all provinces are initially selected as the default setting. Users have the freedom to hand-pick a province of their choice.
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 43 sec

**Review by Jeremy-Prior — APPROVED** [review]
LGTM

**Pull request merged**